cs/src/Contracts/DevTunnels.Contracts.csproj

```diff
@@ -9,11 +9,11 @@
     <UseVsSaaSSdk>false</UseVsSaaSSdk>
     <UseNewtonsoftJson>false</UseNewtonsoftJson>
     <UseVisualStudioValidation>false</UseVisualStudioValidation>
-    <DebugType>embedded</DebugType>
     <GenerateDocumentationFile>True</GenerateDocumentationFile>
     <WarningsNotAsErrors>CS1591</WarningsNotAsErrors>
-    <!-- Include PDB files in NuGet package -->
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+    <!-- Create snupkg for debug symbols: https://learn.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg#creating-a-symbol-package -->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
Should these lines be moved to `cs/src/Directory.Build.props`?

Reply:
If we move these lines to `Directory.Build.props`, we should exclude the test projects from symbol publishing if we don't already do that.

Reply:
Test projects are under a different subdirectory (`cs/test`).

Reply:
Ah, I like that idea - I've moved them to `Directory.Build.props` 😄